### PR TITLE
Add Signoff CI to Enforce Signoff requirement.

### PR DESCRIPTION
**Pull request by FSG-Cat**

### CODE REVIEW

**Comment by Gnuxie** [comment]
Where's your sign off then?

**Comment by FSG-Cat** [comment]
Org member Exception. Yes i should add it give me a second. 

Signed-off-by: Catalan Lover <catalanlover@protonmail.com>

File: .github/workflows/sign-off.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+#  However, this file is modified and the modifications in this file
+#  are NOT distributed, contributed, committed, or licensed under the Apache License.
+#
```
Comment (Gnuxie) [inline .github/workflows/sign-off.yml:26]:
```suggestion
// Copyright 2024 Catalan Lover <catalanlover@protonmail.me>
// Copyright 2022 - 2024 The Matrix.org Foundation C.I.C.
//
// SPDX-License-Identifier: Apache-2.0
//
// SPDX-FileAttributionText: <text>
// This modified file incorporates work from matrix-org/backend-meta
// https://github.com/matrix-org/backend-meta
// </text>
```

Comment (Gnuxie) [reply .github/workflows/sign-off.yml:26]: ^ just do this it's simpler, keep it under apache there's no reason to incorporate this.

**Review by Gnuxie — APPROVED** [review]

**Pull request merged**